core-plugin/src/com/google/gct/idea/git/CloneGcpDialog.java

```diff
@@ -206,4 +208,22 @@ public JComponent getPreferredFocusedComponent() {
   protected JComponent createCenterPanel() {
     return myRootPanel;
   }
+
+  /**
+   * Default dialog state.
+   */
+  private void paintSelectionOK() {
+    myParentDirectory.setBackground(Color.getColor("ECECEC"));
+    parentDirectoryLabel.setForeground(Color.black);
+  }
+
+  /**
+   * Activates when a user selection is incorrect.
+   *
+   * Paints the "Parent Directory" label and textbox background red.
+   */
+  private void paintSelectionError() {
+    myParentDirectory.setBackground(Color.red);
```

Comment:
Color.RED for conformance iwth naming conventions. (Sun got this wrong in the early days and corrected it in 1.4.)